code/datums/diseases/parasitic_infection.dm

```diff
@@ -0,0 +1,48 @@
+/datum/disease/parasite
+	form = "Parasite"
+	name = "Parasitic Infection"
+	max_stages = 4
+	cure_text = "Surgical removal of the liver."
+	agent = "Consuming Live Parasites"
+	spread_text = "Does not spread biologically."
+	viable_mobtypes = list(/mob/living/carbon/human)
+	permeability_mod = 1
+	desc = "If left untreated the subject will passively lose nutrients, and eventually lose their liver."
+	severity = DISEASE_SEVERITY_HARMFUL
+	disease_flags = CAN_CARRY|CAN_RESIST
+	spread_flags = DISEASE_SPREAD_NON_CONTAGIOUS
+	required_organs = list(/obj/item/organ/liver)
+	bypasses_immunity = TRUE
+
+/datum/disease/parasite/stage_act()
+	..()
+	var/mob/living/carbon/C = affected_mob
+	var/obj/item/organ/liver/L = C.getorgan(/obj/item/organ/liver)
+	if(!L)
+		src.cure()
+		C.visible_message("<span class='notice'><B>[C]'s liver is covered in tiny larva! They quickly shrivel and die after being exposed to the open air.</B></span>")
+	switch(stage)
+		if(1)
+			if(prob(5))
+				affected_mob.emote("cough")
+		if(2)
+			if(prob(10))
+				if(prob(50))
+					to_chat(affected_mob, "<span class='notice'>You feel the weight loss already!</span>")
+				affected_mob.adjust_nutrition(-3)
+		if(3)
+			if(prob(20))
+				to_chat(affected_mob, "<span class='notice'>You're... REALLY starting to feel the weight loss.</span>")
+				affected_mob.adjust_nutrition(-6)
```

Comment:
the chat messages are going to be quite spammy at that chance to proc

Reply:
Gotchu fam